zboxcore/sdk/writemarker_mutex.go

```diff
@@ -165,16 +180,27 @@ func (m *WriteMarkerMutex) lockOne(ctx context.Context, buf *bytes.Buffer, url s
 			return err
 		}
 
+		if resp.StatusCode != http.StatusOK {
+			return errors.Throw(constants.ErrNotLockedWritMarker, fmt.Sprint(resp.StatusCode, ":", string(respBody)))
+		}
+
 		err = json.Unmarshal(respBody, result)
 
 		if err != nil {
 			return err
 		}
 
 		return nil
-	})
+	}, resty.WithRetry(resty.DefaultRetry),
```

Comment:
Put these options in a slice and use the slice here to make it easier to read.

Reply:
fixed